tests/maap.cfg

```diff
@@ -0,0 +1,75 @@
+[request]
```

Comment:
I should know this but is there a way to run these tests w/o copying this config file?

Reply:
I don't know. This was necessary to run the data-system-tests notebook, which is a copy of the test notebook you'd created. I never investigated what it was doing, I just saw that it failed without it.

Reply:
ok not worth digging into at this point, IMHO

Reply:
okay. I think the expectation is that all the tests we're writing will superscede the ones in that notebook, so we'll eventually delete it